websocket_server.py

```diff
@@ -0,0 +1,269 @@
+# import asyncio
+import websockets
+import pickle, struct, time, pyaudio
+import threading
+import os
```

Comment:
```suggestion
import os, json
```

Needed to create json string.